### PR TITLE
5.1.0 Release Notes

### PR DESCRIPTION
**Pull request by jeffmaher**

### CODE REVIEW

**Comment by jeffmaher** [comment]
@legostud Ready for your review. Let me know if I missed anything or got something wrong.

File: release-notes.md [hunk 1]
```diff
@@ -1,35 +1,32 @@
...
+- More spacing between patterns has been added.
+- Southbridge location page had its banner coursel replaced with a banner image and Google map.
+- Coloring and spacing fixes on the homepage.
+- Wording and width adjusted on guides.
+- Adjustments to the Location page, with some items removed (quick actions, breadcrumbs, wait times) and added (key actions, contact list).
+- Key actions added to Location Park page.
+- Details content type is more adaptive to different screen sizes.
+- Small label/text changes and other tiny adjustments on Org, G2G, Section Landing, Topic Transition, Helpful Links, Service, Location, and Event Listin pages.
 
 ## 5.0.0
```
Comment (jeffmaher) [inline release-notes.md:32]: Everything below this point is just line endings that my editor or git client must've modified. Not changes to text.

File: release-notes.md [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
+- Event listings and event details for single instance events and those that recur.
+- Directions link now appears after clicking on map pins
+- All available activities at a location are now listed on Location pages
+- Video description/transcript page has been added
+- "Your Government" version of Topic pages now exist
+
+### IMPROVEMENTS
 
+- The main menu (at the top of all pages) is much more accessible with improved ordering if you're tabbing between links.
+- More spacing between patterns has been added.
```
Comment (gender404) [inline release-notes.md]: Consistent spacing between patterns has been added

**Review by gender404 — COMMENTED** [review]
One small change and we're good

File: release-notes.md (hunk 2 above)
Comment (jeffmaher) [reply release-notes.md]: Fixed 😸

**Pull request merged**